symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/UserService.java

```diff
@@ -652,6 +655,33 @@ public FollowingListResponse listUsersFollowing(@Nonnull Long userId, @Nonnull C
     return new PaginatedService<>(api, pagination.getChunkSize(), pagination.getTotalSize()).stream();
   }
 
+  /**
+   * Return a {@link V2UserDetail} with created user details.
+   *
+   * @param payload User's details to create.
+   * @return a {@link V2UserDetail} with created user details.
+   * @see <a href="https://developers.symphony.com/restapi/reference#create-user-v2">Create User v2</a>
+   */
+  @API(status = API.Status.STABLE)
+  public V2UserDetail create(@Nonnull V2UserCreate payload) {
+    return executeAndRetry("createUser",
+        () -> userApi.v2AdminUserCreatePost(authSession.getSessionToken(), payload));
+  }
+
+  /**
+   * Updates an existing V2 Users
+   *
+   * @param userId    User Id
+   * @param payload   User's new attributes for update.
+   * @return {@link V2UserDetail} with updated user details.
+   * @see <a href="https://developers.symphony.com/restapi/reference#update-user-v2">Update User v2</a>
+   */
+  @API(status = API.Status.STABLE)
```

Comment:
Is this annotation something that should be automatically generated (after tests..)  or should it be manually written in the code ?

Reply:
It is manually added but it is useless here as it is put at class level (line 70). This is only to tell our users the API is stable, internal or experimental.